hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/MutableVolumeSet.java

```diff
@@ -522,6 +539,19 @@ private void stopDiskChecker() {
     return ImmutableMap.copyOf(volumeStateMap);
   }
 
+  public boolean hasEnoughVolumes() {
+    int maxVolumeFailuresTolerated = conf.getInt(
+        DFSConfigKeysLegacy.DFS_DATANODE_FAILED_VOLUMES_TOLERATED_KEY,
```

Comment:
DFSConfigKeysLegacy is a deprecated Class.  Since Ozone is spinned out from Hadoop, as a new project, we should avoid use HDFS configuration key anymore.  We can define a similar key for Ozone.

Reply:
@ChenSammi oh, then I shall add this key in some other place.